versions.lock

```diff
@@ -22,7 +22,8 @@ com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.12.3 (5 constraints: b9
 com.fasterxml.jackson.jaxrs:jackson-jaxrs-base:2.12.3 (3 constraints: 903d4249)
 com.fasterxml.jackson.jaxrs:jackson-jaxrs-cbor-provider:2.12.3 (3 constraints: 2d3890f3)
 com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.12.3 (2 constraints: 6e1d2e4b)
-com.fasterxml.jackson.module:jackson-module-afterburner:2.12.3 (10 constraints: 5aa321c1)
+com.fasterxml.jackson.module:jackson-module-afterburner:2.12.3 (9 constraints: 249eb161)
```

Comment:
Are these transitive deps going to be an issue at runtime for downstream consumers? I'm assuming we just have to ensure nothing loads Afterburner and everything migrates to Blackbird?

Reply:
That’s correct, atlas likely hasn’t bumped cjr to a version using bb instead of afterburner, I figured it was best to leave that to excavator.

Reply:
There's going to be some manual work required here since AtlasDB still depends on some `remoting2` packages which pull in afterburner.
```
com.fasterxml.jackson.module:jackson-module-afterburner:2.12.3
        com.fasterxml.jackson:jackson-bom -> 2.12.3
        com.palantir.conjure.java.runtime:conjure-java-jackson-serialization -> 2.12.3
        com.palantir.conjure.java.runtime:conjure-java-jersey-server -> 2.12.3
        com.palantir.remoting2:error-handling -> 2.6.7
        com.palantir.remoting2:jackson-support -> 2.6.7
        com.palantir.remoting2:jersey-servers -> 2.6.7
        com.palantir.remoting2:tracing -> 2.6.7
        com.palantir.tracing:tracing -> 2.10.4
        io.dropwizard:dropwizard-jackson -> 2.10.2
```